src/consensus/validation.h

```diff
@@ -22,6 +22,39 @@ static const unsigned char REJECT_NONSTANDARD = 0x40;
 static const unsigned char REJECT_INSUFFICIENTFEE = 0x42;
 static const unsigned char REJECT_CHECKPOINT = 0x43;
 
+/** A "reason" why something was invalid, suitable for (possibly) getting angry
+  * (and retaliating) at the provider of the object in question (ie banning
+  * peers).
+  * These are much more granular than the rejection codes, which may be more
+  * useful for some other use-cases.
+  */
+enum class ValidationInvalidReason {
```

Comment:
Renaming this `InvalidReason` would save a lot of chars overall. The only other prefix in this module is `REJECT_`